tests/frontend/components/test_build_edit_section.py

```diff
@@ -0,0 +1,31 @@
+"""Test function build_edit_section."""
+
+import pytest
+from datadoc_model import model
+
+from datadoc.frontend.components.builders import build_edit_section
+from datadoc.frontend.fields.display_variables import OBLIGATORY_VARIABLES_METADATA
+
+NORSK_BOKMÅL = "nb"
```

Comment:
I believe we already have these constants in `tests/utils.py`, we should just use those ones.

Reply:
I can't find languages in tests/utils.py. Maybe I should just import/use SupportedLanguages.NORSK_BOKMÅL ?